gce/makeProject

```diff
@@ -0,0 +1,246 @@
+#!/bin/bash
+#
+#    Copyright 2016 Google, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+set -ex
+
+BUCKET=
+
+# [START useful]
```

Comment:
This region tag doesn't look like it's meeting the requirements for the sample tracker.

Reply:
Same of the other region tags in this file.